sdk/storage/azure-storage-blob-changefeed/samples/change_feed_samples.py

```diff
@@ -87,16 +89,76 @@ def list_events_using_continuation_token(self):
         token = change_feed.continuation_token
 
         # restart using the continuation token
-        change_feed2 = cf_client.list_changes(results_per_page=3).by_page(continuation_token=token)
+        change_feed2 = cf_client.list_changes(results_per_page=56).by_page(continuation_token=token)
         change_feed_page2 = next(change_feed2)
         for event in change_feed_page2:
             print(event)
 
+    def list_events_in_live_mode(self):
+        # Instantiate a ChangeFeedClient
+        cf_client = ChangeFeedClient("https://{}.blob.core.windows.net".format(self.ACCOUNT_NAME),
+                                     credential=self.ACCOUNT_KEY)
+        # to get continuation token
+        start_time = datetime(2020, 8, 19, 10)
+        change_feed = cf_client.list_changes(start_time=start_time).by_page()
+
+        for page in change_feed:
+            for event in page:
+                print(event)
+        token = change_feed.continuation_token
+
+        sleep(120)
+        print("continue printing events")
+        # restart using the continuation token
+        change_feed2 = cf_client.list_changes(results_per_page=56).by_page(continuation_token=token)
+        change_feed_page2 = next(change_feed2)
+        for event in change_feed_page2:
+            print(event)
+
+        sleep(120)
+        print("continue printing events")
+
+        token2 = change_feed2.continuation_token
+        change_feed3 = cf_client.list_changes(results_per_page=56).by_page(continuation_token=token2)
+        change_feed_page3 = next(change_feed3)
+        for event in change_feed_page3:
+            print(event)
+
+    def list_events_in_live_mode_continuously_without_waiting_for_a_minute(self):
```

Comment:
I think we should remove this example. afaik CF can flush events as fast as in 1 or 2 minutes interval. So customer shouldn't be really polling more frequently than once per minute, otherwise it's gonna be waste of resources and money. We should direct them towards polling per minute or longer interval.